repondeur/README.md

```diff
@@ -11,6 +11,30 @@ $ pipenv install
 $ pipenv shell
 ```
 
+## Initialize the database
+
+```
+$ zam_init_db development.ini
+```
+
+## Extra setup
+
+Get the groups data for Assemblée nationale:
+
+```
+$ curl --silent --show-error http://data.assemblee-nationale.fr/static/openData/repository/15/amo/deputes_actifs_mandats_actifs_organes_divises/AMO40_deputes_actifs_mandats_actifs_organes_divises_XV.json.zip -o groups.zip
+$ mkdir -p data/an/groups
+$ unzip -q -o groups.zip -d data/an/groups/
+$ rm groups.zip
+```
+
+Define the folowing environment variables:
```

Comment:
`folLowing`

Reply:
Fixed in 9d5c65d8442b943a3c09d432d52a6645a99ac6f2